terraform/admin/ci.tf

```diff
@@ -84,7 +84,7 @@ resource "aws_codebuild_project" "pr_ci" {
 
     environment_variable {
       name  = "TF_CLI_ARGS"
-      value = "-no-color"
+      value = "-no-color -lock-timeout=${var.terraform_lock_timeout_duration}"
```

Comment:
I can't tell if this is the best place to do this or if I should just expose an env var called TF_LOCK_TIMEOUT_DURATION and wire it up in ci and cd scripts. @tptee what do you think?

Reply:
@tptee -- want to take a stab at your thoughts here? Thanks!